browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -277,16 +302,6 @@ void BraveTabContainer::CompleteAnimationAndLayout() {
   base::ranges::for_each(children(), &views::View::Layout);
 }
 
-void BraveTabContainer::OnPaintBackground(gfx::Canvas* canvas) {
-  if (!tabs::utils::ShouldShowVerticalTabs(
-          tab_slot_controller_->GetBrowser())) {
-    TabContainerImpl::OnPaintBackground(canvas);
-    return;
-  }
-
-  canvas->DrawColor(GetColorProvider()->GetColor(kColorToolbar));
-}
-
```

Comment:
Right - it now paints the background in `BraveTabStrip`, because `BraveTabStrip` paints to an opaque layer. Since `BraveTabStrip` is a parent of `BraveTabContainer`, the background painting in `BraveTabContainer` is no longer required.